crates/precompile/Cargo.toml

```diff
@@ -46,6 +46,13 @@ cometbft-proto = {  git = "https://github.com/bnb-chain/greenfield-cometbft-rs.g
 cometbft-light-client = {  git = "https://github.com/bnb-chain/greenfield-cometbft-rs.git", rev = "1282547" }
 prost = { version = "0.12.3" }
 bls_on_arkworks = "0.3.0"
+tendermint = { git = "https://github.com/forcodedancing/tendermint-rs-parlia", rev = "10cfd14bac4cd4f1180a6670ea44edabfeda2519", features = ["secp256k1"] }
```

Comment:
`forcodedancing`?  We need an official repo

Reply:
Fixed